charts/alertmanager/templates/alertmanager-networkpolicy.yaml

```diff
@@ -26,12 +26,22 @@ spec:
           tier: monitoring
           component: prometheus
           release: {{ .Release.Name }}
+    {{- if .Values.global.sidecar.enabled  }}
```

Comment:
```suggestion
    {{- if .Values.global.authSidecar.enabled  }}
```

Reply:
Im have modified the entire chart doing on round of testing - will push with updated one.

Reply:
sidecar name moved to authSidecar